lib/serializer.js

```diff
@@ -5,16 +5,22 @@ const pb = require('protobufjs')
 
 module.exports.init = function (apiFile/* : string */) {
   return pb.load(apiFile || './lib/api.proto').then((r) => {
-    return new Serializer({
+    if (this.serializer) { return this.serializer }
```

Comment:
i'm not sure this line helps actually. https://nodejs.org/api/modules.html#modules_caching suggests that every module import gets the same object, so calling 'init' won't help future imports.